openedx/features/idea/helpers.py

```diff
@@ -0,0 +1,14 @@
+def upload_to_path(instance, filename, folder):
```

Comment:
I'm not sure but i feel like I've seen this method somewhere else. can you confirm?

Reply:
Yes, but we cannot use it here. There are one or two relevant methods but they are in course discovery repo, and they need some tweaks

Reply:
okay